.devcontainer/post-create.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+(cd discussions && ./build.sh)
+(cd community && ./build.sh)
+(cd community-factory && ./build.sh)
+./build.sh
```

Comment:
What if we add building discussions, community and community-factory in root build.sh and call just this one? @frol

Reply:
1. There are no `build.sh` scripts anymore
2. We need to implement `build.rs` that would build the dependency contract
3. Let's use `ghcr.io/near/near-devcontainer:latest` image for the container, which already has all the NEAR CLIs; you can test the container on https://github.com/near/cargo-near-new-project-template repo

Reply:
I've changed to `ghcr.io/near/near-devcontainer:latest`, and `build.sh` is not used anymore. Instead `cargo near build` is invoked from all the subcontract folders, like in the build pipeline.

I did try implementing a `build.rs` like below, but I gave up because of settings/env propagating from the parent build, and too much too hack into right now.

```rust
use std::process::Command;
use std::env;
use std::path::Path;

fn main() {
    let projects = ["discussions", "community", "community-factory"]; 

    for project in projects {
        let project_path = if project == "." {
            env::var("CARGO_MANIFEST_DIR").unwrap()
        } else {
            Path::new(&env::var("CARGO_MANIFEST_DIR").unwrap())
                .join(project)
                .to_str()
                .unwrap()
                .to_string()
        };

        let mut command = Command::new("cargo");
        command.arg("near").arg("build")
               .current_dir(project_path);

        let status = command.status().expect(&format!("Failed to execute cargo near build in {}", project));

        if !status.success() {
            panic!("Failed to build NEAR contract in project: {}", project);
        }
    }
}
```